docs/packages/dev-workflow.md

```diff
@@ -2,107 +2,175 @@
 
 This document provides guidance on how to build and publish package and repo bundles and test them on your local cluster.
 
-1. Build CLI plugins
+> For detailed info about variables and targets used here, see [build-tooling](https://github.com/vmware-tanzu/build-tooling-for-integrations/blob/main/docs/build-tooling-getting-started.md).
 
-   CLI plugins need to be built by running the below make target, because they are used to generate
-   `core-management-plugins` and `standalone-plugins` package bundle. There is WIP to remove this step and build each
-   package bundle individually.
+1. Build the container images for all the packages
 
-   ```shell
-      make build-cli
-   ```
-
-2. Build the container images for all the packages
+   Images need to be built by running the below make target, because they are used to generate
+   package bundles.
 
    ```shell
-      OCI_REGISTRY=${REGISTRY} make docker-all
+      OCI_REGISTRY="****" make docker-build-all -f build-tooling.mk
    ```
 
-   This would build, push a docker image for the component and also replaces the newImage path in the [kbld-config.yaml](../../packages/tkg-autoscaler/kbld-config.yaml)
-
-3. Start local registry
-
-   Local docker registry is used for pushing the package bundle to get the sha256, to use it later when producing repo bundle.
-
-   Run the below make target to start the local registry
+2. Publish the container images for all the packages
 
    ```shell
-      make local-registry
+      OCI_REGISTRY="****" make docker-publish-all -f build-tooling.mk
    ```
+   This would push contianer images for the components to specified registry and also replaces the newImage path in the [kbld-config.yaml](../../packages/readiness/kbld-config.yaml)
 
-4. Build the package bundles that belong to a particular package repository
+3. Build the package bundles that belong to a particular package repository
 
-   To build the package bundles for all the packages in the management package repository run:
+   To build the package bundles for all the packages in the `management` package [repository](../../packages/package-values.yaml) run:
```

Comment:
Package repository references can be renamed from `management` to `runtime-core` in this doc.

Reply:
Updated